src/ControlSystem/Actions/Initialization.hpp

```diff
@@ -41,13 +47,18 @@ struct InitializeControlSystem {
   // at the moment, because this is the DerivOrder of most functions of time in
   // the domain creators. If we want to choose the DerivOrder at runtime (i.e.
   // in an input file), that capability will need to be added later.
-  using initialization_tags = tmpl::list<control_system::Tags::Averager<2>,
-                                         control_system::Tags::TimescaleTuner>;
+  static constexpr size_t deriv_order = ControlSystem::deriv_order;
```

Comment:
Update the comment above

Reply:
Removed it because it no longer applied here :)